create_test_data.py

```diff
@@ -0,0 +1,91 @@
+"""
+
+This is a script for generating test holdshelf events in the local psql db.
+
+Usage:
+    AWS_PROFILE=nypl-digital-dev ENVIRONMENT=development \
+        python create_test_data.py
+
+"""
+
+import os
+import datetime
+
+from nypl_py_utils import PostgreSQLClient, PostgreSQLClientError
+from nypl_py_utils.functions.config_helper import load_env_file
+
+HOLDS = [
+    [1, 10001, 'i', 'YYYY-MM-DDT08:11:01', 'mal'],
+    [2, 10002, 'i', 'YYYY-MM-DDT08:31:01', 'mal']
+]
+
+ITEMS = [
+    (10001, 101, 'maf92'),
+    (10002, 102, 'maf92')
+]
+
+sierra_client = None
+
+
+def connect():
+    global sierra_client
+
+    load_env_file(os.environ['ENVIRONMENT'], 'config/development.yaml')
+    sierra_client = PostgreSQLClient(
+        '127.0.0.1',
+        os.environ['SIERRA_DB_PORT'],
+        os.environ['SIERRA_DB_NAME'],
+        os.environ['SIERRA_DB_USER'],
+        os.environ['SIERRA_DB_PASSWORD']
+    )
+    sierra_client.connect()
+
+
+def disconnect():
+    global sierra_client
+    sierra_client.close_connection()
+
+
+def insert_data(table, row):
+    global sierra_client
+
+    values_str = ', '.join(['%s' for n in range(len(row))])
+    query = 'INSERT INTO sierra_view.{} VALUES ({})'.format(table, values_str)
+    print(' => {} < {}'.format(query, row))
+    db_query(query, row)
+
+
+def db_query(query, data=None):
```

Comment:
What's the difference between this method and the existing PostgreSQLClient query() method?